cypress/integration/no-toolbar.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2018-2021 Elyra Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+describe("no toolbar", () => {
+  before(() => {
+    cy.visit("/iframe.html?id=example-pipelineeditor--no-toolbar");
+  });
+
+  it("renders empty pipeline message", () => {
+    cy.findByText(/your flow is empty/i).should("exist");
```

Comment:
Should it test that there isn't a toolbar?

Reply:
I was just using no-toolbar as a codename for vscode haha. I just wanted to have one working cypress test at least, I'll add more extensive tests in a follow-up PR

Reply:
Ok sounds good